src/views/statusView.ts

```diff
@@ -217,6 +226,19 @@ export default class StatusView implements vscode.Disposable {
         this.showStatusBarItem(fileUri, bar.sqlCmdMode);
     }
 
+    public showRowCount(fileUri: string, message?: string): void {
+        let bar = this.getStatusBar(fileUri);
+        if (message) {
+            bar.rowCount.text = message.slice(1, -1);
```

Comment:
Might want to explain why you're doing this - I assume because there's a leading space or something?

Reply:
Added comment